CONTRIBUTING.md

```diff
@@ -6,13 +6,75 @@ Microsoft Authentication Library (MSAL) for .NET welcomes new contributors.  Thi
 
 Please visit [https://cla.microsoft.com/](https://cla.microsoft.com/) and sign the Contributor License Agreements.  You only need to do that once. We can not look at your code until you've submitted this request.
 
-## Tests
+## Contributing your own code changes
```

Comment:
@localden Thoughts on moving the section on Submitting a bug/feature request above this one?